docs/config_search.md

```diff
@@ -150,7 +151,19 @@ as the range for the `max_batch_size` to `[1]`, it will no longer be a valid
 Triton Model Configuration.
 
 The configuration sweep described above, will sweep through 8 configs = (2
-`max_batch_size`) * (2 `max_queue_delay_microseconds`) * (2 `instance_group`) values.
+`max_batch_size`) _ (2 `max_queue_delay_microseconds`) _ (2 `instance_group`) values.
+
+## Quick Search
+
+Quick search is enabled by adding the parameter `--run-config-search-mode quick` to the CLI.
+
+It uses a hill climbing algorithm to search the configuration space, looking for
+the maximal objective value within the specified constraints. In the majority of cases
+this will find greater than 90% of the maximum objective value (that could be found using a brute force search),
+while needing to search less than 10% of the configuration space.
+
+It will then sweep the top-N configurations found (specified by `--num-configs-per-model`) over the default concurrency
+range before generation of the summary reports.
 
 ### Examples of Additional Model Config Parameters
```

Comment:
This section should stay part of manual brute section

Reply:
Fixed